src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -407,6 +513,20 @@ def exec_aggregation(
         run_id=None,
         node_concurrency=DEFAULT_CONCURRENCY_FLOW,
     ) -> AggregationResult:
+        """Execute the aggregation node of the flow.
+
+        :param inputs: A mapping of input names to their values.
+        :type inputs: Mapping[str, Any]
+        :param aggregation_inputs: A mapping of aggregation input names to their values.
+        :type aggregation_inputs: Mapping[str, Any]
+        :param run_id: The ID of the current run, if any.
+        :type run_id: Optional[str]
+        :param node_concurrency: The maximum number of nodes that can be executed concurrently.
+        :type node_concurrency: int
+        :return: The result of the aggregation node.
+        :rtype: AggregationResult
```

Comment:
is this promptflow defined type?

Reply:
Add in full package style. But that's one module in _result, so still we can't have link for this case.